scripts/create-icons.js

```diff
@@ -30,11 +31,31 @@ function generateIcons() {
     console.log('Generating icon.ico');
     const icoIcon = png2icons.createICO(pngBuffer, png2icons.BICUBIC2, 0, false, true);
     fs.writeFileSync(icoOutPath, icoIcon);
+  } else {
+    fs.cpSync(icoPath, icoOutPath);
```

Comment:
This was just an oversight that it wasn't there before. Unrelated to the functionality of the systray.